plugins/ggdm30_rules.js

```diff
@@ -4018,7 +4018,8 @@ ggdm30.rules = {
     ['RLE','1','layer','1'], // Raised
     ['RLE','2','layer','0'], // Level with surrounding area
     ['RLE','3','layer','-1'], // Depressed
-    ['RLE','998',undefined,undefined], // Not Applicable
+    // ['RLE','998',undefined,undefined], // Not Applicable
+    ['RLE','998','relative_level','not_applicable'], // Not Applicable
```

Comment:
Do we really need the old value commented out if it is good enough to go into production?